faker/providers/person/hy_AM/__init__.py

```diff
@@ -0,0 +1,985 @@
+# coding=utf-8
+
+from __future__ import unicode_literals
+from .. import Provider as PersonProvider
+
+
+class Provider(PersonProvider):
+    formats_male = (
+        '{{first_name_male}} {{last_name}}',
+    )
+
+    formats_female = (
+        '{{first_name_female}} {{last_name}}',
+    )
+
+    formats = formats_male + formats_female
+
+    first_names_male = (
```

Comment:
could you include the source for this dataset?

Reply:
I've add the source for name datasets, thank you for point this out.